clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -319,10 +319,10 @@ Changes in existing checks
   diagnostic.
 
 - Improved :doc:`readability-implicit-bool-conversion
-  <clang-tidy/checks/readability/implicit-bool-conversion>` check
-  by adding the option `UseUpperCaseLiteralSuffix` to select the
-  case of the literal suffix in fixes and fixing false positive for implicit
-  conversion of comparison result in C23.
+  <clang-tidy/checks/readability/implicit-bool-conversion>` 
+   - `UseUpperCaseLiteralSuffix` : check by adding the option to select the case of the literal suffix in fixes and fixing false positive for implicit conversion of comparison result in C23.
```

Comment:
`check by adding the options` should be common for all three options.

Reply:
Sorry, if I was not clear, `check by adding the options` should be added once before list of added options.

Reply:
Thanks I have done it in Release Notes .

Reply:
It is not necessary to repeat `check by adding the option` thrice.

Reply:
Its looking good ?